include/ultra64/gbi.h

```diff
@@ -10,14 +10,6 @@
 /* Private macro to wrap other macros in do {...} while (0) */
 #define _DW(macro) do {macro} while (0)
```

Comment:
eventually we'll need a way to turn this into `{ macro } (void)0` or something. Should we do this now or later? I'm not sure how often we plan to sync the two

Reply:
Added a `GBI_DOWHILE` toggle for this